test/server/pglib.py

```diff
@@ -35,6 +35,31 @@ def attach_device(device_id, account_id):
     return {"accountid": account_id, "accountid.sig": account_id_hmac}
 
 
+def get_device_id(account_id):
+    dbname = pg_vars["PGDATABASE"] or pg_vars["PGUSER"]
+    conn = psycopg2.connect(dbname=dbname, user=pg_vars["PGUSER"], host=pg_vars["PGHOST"], port=pg_vars["PGPORT"])
+    conn.autocommit = True
+    cur = conn.cursor()
+    cur.execute("SELECT id FROM devices WHERE accountid = '" + account_id + "'")
+    data = cur.fetchall()
+    cur.close()
+    conn.close()
+    return {"deviceid": row[0] for row in data}
```

Comment:
Shouldn't this be `[row[0] for row in data]` ? I don't think this should return a dictionary.

Reply:
row[0] for row in data is returning string here